src/sdf.rs

```diff
@@ -24,7 +29,7 @@ pub fn march_circle<F: Fn(Real2)->Real>(start: Real2, end: Real2, radius: Real,
     let mut l = reali(0);
     let mut hit = false;
     let forward_step = reali(1);
-    let backward_step = Real::from(0.125);
```

Comment:
Is this a manual change or automatic clippy change?
I intend to build this with Real being a type-def of a fixed point type, and then I will need similar conversions of constants...

Reply:
This is an automatic clippy change. If you implement the `From` trait those conversions will be done automatically right?

Reply:
I thought one have to call .into() or Real::from() explicitly, but I may be wrong.

Reply:
I will take this and just replace it with a function later so it does not mess with clippy.